### PR TITLE
Block curl revisions which do not build

### PR DESCRIPTION
**Pull request by Sinerum**

### CODE REVIEW

**Review by vulder — APPROVED** [review]

File: varats/varats/projects/c_projects/curl.py [hunk 1]
```diff
@@ -32,13 +33,22 @@ class Curl(VProject):
...
-            refspec="origin/HEAD",
-            limit=None,
-            shallow=False
+        block_revisions([
+            GoodBadSubgraph(["3af90a6e19249807f99bc9ee7b50d3e58849072a"],
+                            ["30ef1a077996c71e463beae53354e8ffc7a4c90d"],
+                            "compile error without ssl"),
+            GoodBadSubgraph(["ae1912cb0d494b48d514d937826c9fe83ec96c4d"],
+                            ["98dcde4ec3397d8626e2c8f29abaf481fc42e8ec"],
+                            "Checks for old libtool version")
```
Comment (boehmseb) [inline varats/varats/projects/c_projects/curl.py]:
```suggestion
                            "Requires old libtool version")
```

File: varats/varats/projects/c_projects/curl.py [hunk 2]
```diff
@@ -32,13 +33,22 @@ class Curl(VProject):
...
-            project_name="curl",
-            remote="https://github.com/curl/curl.git",
-            local="curl",
-            refspec="origin/HEAD",
-            limit=None,
-            shallow=False
+        block_revisions([
+            GoodBadSubgraph(["3af90a6e19249807f99bc9ee7b50d3e58849072a"],
+                            ["30ef1a077996c71e463beae53354e8ffc7a4c90d"],
+                            "compile error without ssl"),
```
Comment (boehmseb) [inline varats/varats/projects/c_projects/curl.py]:
```suggestion
                            "Compile error without ssl"),
```

File: varats/varats/projects/c_projects/curl.py (hunk 1 above)
Comment (boehmseb) [inline varats/varats/projects/c_projects/curl.py]:
```suggestion
                            "Requires old libtool version")
```

**Review by boehmseb — APPROVED** [review]
Capitalization in messages is inconsistent. Otherwise looks good.

**Pull request merged**